src/style.css

```diff
@@ -0,0 +1,51 @@
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: url("images/star-active.svg");
+  background-size: auto;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(1),
+.stars--0 .stars__star:nth-child(2),
+.stars--0 .stars__star:nth-child(3),
+.stars--0 .stars__star:nth-child(4),
+.stars--0 .stars__star:nth-child(5) {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(2),
+.stars--1 .stars__star:nth-child(3),
+.stars--1 .stars__star:nth-child(4),
+.stars--1 .stars__star:nth-child(5) {
+  background-image: url(images/star.svg);
+}
+
+.stars--2 .stars__star:nth-child(3),
+.stars--2 .stars__star:nth-child(4),
+.stars--2 .stars__star:nth-child(5) {
+  background-image: url(images/star.svg);
+}
+
+.stars--3 .stars__star:nth-child(4),
+.stars--3 .stars__star:nth-child(5) {
+  background-image: url(images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(5) {
+  background-image: url(images/star.svg);
+}
```

Comment:
It looks quite complicated and a lot of code is duplicated, you need to simplify it and combine the selectors with a comma